search/lib/convert/granule-to-item.js

```diff
@@ -31,20 +26,20 @@ const cmrBoxToGeoJsonPolygon = (box) => {
       [w, s]
     ]]
   };
-};
+}
 
-// Move to granule_to_item
-const cmrSpatialToGeoJSONGeometry = (cmrGran) => {
+function cmrSpatialToGeoJSONGeometry (cmrGran) {
   let geometry = [];
   if (cmrGran.polygons) {
     geometry = geometry.concat(cmrGran.polygons.map(cmrPolygonToGeoJsonPolygon));
   }
   if (cmrGran.boxes) {
     geometry = geometry.concat(cmrGran.boxes.map(cmrBoxToGeoJsonPolygon));
   }
+  //Example: points: ['12,34', '45,67']
   if (cmrGran.points) {
     geometry = geometry.concat(cmrGran.points.map((ps) => {
-      const [lat, lon] = parseOrdinateString(ps);
+      const [lon, lat] = parseOrdinateString(ps);
```

Comment:
Is this what is expected from a point?

Reply:
I would think that a point string, [10, 5], would be in the form [latitude, longitude].

Reply:
@PeteShepley I found this a little odd as well, but everything we could find online in the STAC Item Spec, CMR docs, and searches on GeoJSONs had it ordered as [longitude, latitude].

Reply:
Fair. I trust you.